src/Uno.UWP/UI/Composition/CompositionGradientBrush.cs

```diff
@@ -0,0 +1,109 @@
+﻿#nullable enable
+
+using System;
+using System.Numerics;
+
+namespace Windows.UI.Composition
+{
+	public partial class CompositionGradientBrush : CompositionBrush
+	{
+		private CompositionGradientExtendMode _extendMode;
+		private CompositionMappingMode _mappingMode;
+		private Matrix3x2 _transformMatrix = Matrix3x2.Identity;
+		private Matrix3x2 _relativeTransformMatrix = Matrix3x2.Identity;
+		private Vector2 _scale = new Vector2(1, 1);
+		private float _rotationAngleInDegrees;
+		private float _rotationAngle;
+		private Vector2 _offset;
+		private Vector2 _centerPoint;
+
+		internal CompositionGradientBrush(Compositor compositor)
+			: base(compositor)
+		{
+			ColorStops = new CompositionColorGradientStopCollection(this);
+		}
+
+		public CompositionColorGradientStopCollection ColorStops { get; }
+
+		public CompositionGradientExtendMode ExtendMode
+		{
+			get => _extendMode;
+			set => SetProperty(ref _extendMode, value); 
+		}
+
+		public CompositionMappingMode MappingMode
+		{
+			get => _mappingMode;
+			set => SetProperty(ref _mappingMode, value);
+		}
+
+		public Matrix3x2 TransformMatrix
+		{
+			get => _transformMatrix;
+			set => SetProperty(ref _transformMatrix, value);
+		}
+
+		public Vector2 Scale
+		{
+			get => _scale;
+			set => SetProperty(ref _scale, value);
+		}
+
+		public float RotationAngleInDegrees
+		{
+			get => _rotationAngleInDegrees;
+			set { _rotationAngle = value * (float)(Math.PI / 180); SetProperty(ref _rotationAngleInDegrees, value); }
```

Comment:
```suggestion
			set 
			{ 
			   _rotationAngle = value * (float)(Math.PI / 180);
			   SetProperty(ref _rotationAngleInDegrees, value); 
			}
```

Reply:
Changed for both RotationAngle and RotationAngleInDegrees setters. 👍